File: Taller de Haskell/Haskell/problem7.hs
-- Evaluación de los empleados
-- <5 = Inaceptable ($0)
-- 5 = Aceptable ($30000)
-- >=8 = Meritorio ($60000)

evaluarEmpleado :: Int -> (String, Int)
evaluarEmpleado puntuacion
    | puntuacion < 5 = ("Inaceptable", 0)
    | puntuacion == 5 = ("Aceptable", 30000)
    | otherwise = ("Meritorio", 60000)

main :: IO ()
main = do
    putStrLn "Ingrese la puntuacion: "
    input <- getLine
    let puntuacion = read input :: Int
    let (nivel, recompensa) = evaluarEmpleado puntuacion
    putStrLn ("Nivel: " ++ nivel ++ ", Recompensa monetaria: $" ++ show recompensa)

File: Taller de Haskell/Python/problem7.py
# Evaluación de los empleados

# Criterio:
# <5 = Inaceptable ($0)
# 5 = Aceptable ($30000)
# >= 8 = Meritorio ($60000)

puntuacion = int(input("Ingrese la puntuacion: "))

if puntuacion < 5:
    nivel = "Inaceptable"
    recompensa = 0
elif puntuacion == 5:
    nivel = "Aceptable"
    recompensa = 30000
else:
    nivel = "Meritorio"
    recompensa = 60000

print(f"Nivel: {nivel}, Recompensa monetaria: ${recompensa} ")